sdk/python/feast/ui/package.json

```diff
@@ -6,7 +6,7 @@
     "@elastic/datemath": "^5.0.3",
     "@elastic/eui": "^55.0.1",
     "@emotion/react": "^11.9.0",
-    "@feast-dev/feast-ui": "0.29.0",
+    "@feast-dev/feast-ui": "0.33.1",
```

Comment:
this seems inconsistent

Reply:
oh maybe the UI version is different than the python sdk? i assumed this should be 0.28 because of our branch